app/__init__.py

```diff
@@ -13,6 +13,7 @@
 from importlib import import_module
 from flask_migrate import Migrate
 from flask_jwt_extended import JWTManager
+from elasticsearch import Elasticsearch
```

Comment:
I thought we were to use the `elasticsearch_dsl` SDK for this

Reply:
I have updated the code to use `elasticsearch_dsl` SDK.